### PR TITLE
Improve basic development experience

### PR DESCRIPTION
**Pull request by mdziekon**
Changelog:
- Added support for switching dev server port
- Added support for including one single device to the dev build, even in `EXCLUDE_DEVICES=yes` mode
- Added `.nvmrc` file for nvm or nvm-compatible tool users, with node version conforming to the README.md docs

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16
```
Comment (Koenkk) [inline .nvmrc:1]: I don't expect any problem, but could you check if node 18 also works fine (node 16 will go EOL in +- 7 months from now)

Comment (Koenkk) [reply .nvmrc:1]: Preferably in both places, or maybe even better, can `.nvmrc` be removed such that you are forced to specify it?

Comment (mdziekon) [reply .nvmrc:1]: I'm not sure if I follow - how removing the `.nvmrc` file is better? `.nvmrc`'s purpose is to specify that version for you, similar to how `.editorconfig`'s file is to specify IDE's / editor's configuration.

**Comment by Koenkk** [comment]
Looks good now, thanks!

**Pull request merged**